lightspeed/assemblies/assembly_logging-in-out-portal.adoc

```diff
@@ -9,22 +9,24 @@ ifdef::context[:parent-context: {context}]
 :context: logging-in-out-lightspeed-portal
 
 [role="_abstract"]
-Ansible users can use {LightspeedShortName} after the organization administrator has assigned them seat licenses. You access Ansible Lightspeed through the Ansible Lightspeed portal. {LightspeedShortName} uses {RHSSO} ({RHSSOshort}) for authentication, and verifies that you have an assigned seat license. After you log in to the Ansible Lightspeed portal, your assigned user role is displayed on the login screen. 
+
+By default, all Ansible users within an organization that has a trial or paid subscription to both {PlatformNameShort} and {ibmwatsonxcodeassistant} can access {LightspeedShortName}.
+
+You can access Ansible Lightspeed through the link:https://c.ai.ansible.redhat.com/[Ansible Lightspeed portal]. After you enter your {RHSSO} ({RHSSOshort}) account credentials, your account is authenticated and access is granted. Your assigned user role is displayed on the login screen of Ansible Lightspeed portal. 
```

Comment:
To avoid passive voice, change to "your account is authenticated and you are granted access"

I would also add an article to Ansible Lightspeed portal so that it reads "**the** Ansible Lightspeed portal"

Reply:
@hherbly : I addressed this comment while I was at it.